src/main/java/io/github/thunderz99/cosmos/condition/Condition.java

```diff
@@ -214,6 +217,17 @@ public Condition join(Set<String> join) {
         return this;
     }
 
+    /**
+     * If it is true, return all the result in the sub array.
+     *
+     * @param isReturnAllSubArray default value is true
+     * @return condition
+     */
+    public Condition isReturnAllSubArray(boolean isReturnAllSubArray) {
```

Comment:
`isReturnAllSubArray` -> `returnAllSubArray`

Please add a description in comment, that this function works only when `join` is 
 used.